### PR TITLE
INS 21572 role based access to KC via APIv2

### PR DESCRIPTION
**Pull request by jacobhuanginstaclustr**

### CODE REVIEW

**Review by debaditya-netapp — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 BIN_NAME=terraform-provider-instaclustr
 
-VERSION=2.0.15
+VERSION=1.29.1
```
Comment (arjunrajnetapp) [inline Makefile:3]: Should be v2.0.16

**Review by arjunrajnetapp — CHANGES_REQUESTED** [review]

**Review by WeiXinFam-Insta — COMMENTED** [review]
Where's the info on if no access key/secret key nor iam role provided, will be the attach permission policy method by default?

**Comment by jacobhuanginstaclustr** [comment]
> 

It is included in the resource documentation.

File: Makefile (hunk 1 above)
Comment (jacobhuanginstaclustr) [reply Makefile]: Fixed it :)

**Review by arjunrajnetapp — APPROVED** [review]

**Pull request merged**